src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -47,9 +47,17 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
     return true;
   };
 
+  window.addEventListener('unload', sendDisconnect, false);
+
   window['Blazor'].reconnect = reconnect;
 
   logger.log(LogLevel.Information, 'Blazor server-side application started.');
+
+  function sendDisconnect(_: Event): void {
+    const data = new FormData();
+    data.set('circuitId', circuit.circuitId);
```

Comment:
Not sure what browser versions you want to support, because `set` is actually quite _new_ and `append` would also work in this case.
https://developer.mozilla.org/en-US/docs/Web/API/FormData/append#Browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/API/FormData/set#Browser_compatibility

(Mabye also add a check for `sendBeacon`?)

Reply:
set is supported on all browsers we care about. I will change it to append though because its trivial.

sendBeacon is also supported on all browsers we care about, and even if that's not the case you would get an error when you are unloading the document, so that's not a problem.